include/modules/memory.hpp

```diff
@@ -31,6 +32,10 @@ namespace modules {
     progressbar_t m_bar_memfree;
     int m_perc_memused{0};
     int m_perc_memfree{0};
+    int m_perc_memavail{0};
```

Comment:
I think we shouldn't have two variables storing the same thing. We should keep the new ones and removed the old ones.

Reply:
Would this not be considered a breaking change?

Reply:
I just want to remove the variables `m_perc_memused` and `m_perc_memfree`, not the labels ^^